Assets/Scripts/DataTypes/Vertices/CrossSection.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace DataTypes
+{
+    public class CrossSection : Vertex
+    {
+        protected Anchor upAnchor;
+        protected Anchor rightAnchor;
+        protected Anchor downAnchor;
+        protected Anchor leftAnchor;
```

Comment:
Ist die Orientierung der ein-/ausgehenden Straßen relevant?

Reply:
Nein?
Straßen umfassen Spuren in beide Richtungen.